cli/lib/kontena/cli/stacks/logs_command.rb

```diff
@@ -9,13 +9,20 @@ class LogsCommand < Kontena::Command
     banner "Shows logs from services in a stack"
 
     parameter "NAME", "Stack name"
+    parameter "[SERVICE] ...", "Service names"
```

Comment:
That's quite difficult to use in scripts and tedious to type. 

Stack monitor is consistent with this:
```
Usage:
    kontena stack monitor [OPTIONS] NAME [SERVICES] ...

  Monitor services in a stack

Parameters:
    NAME                          Stack name
    [SERVICES] ...                Stack services to monitor
```